config/metadata/metadata.yaml

```diff
@@ -1,6 +1,9 @@
 apiVersion: clusterctl.cluster.x-k8s.io/v1alpha3
 kind: Metadata
 releaseSeries:
+- major: 0
+  minor: 3
+  contract: v1alpha3
```

Comment:
should we replace the `0.2` block? or is this fine?

Reply:
https://github.com/talos-systems/cluster-api-bootstrap-provider-talos/pull/23/files

Reply:
normal to keep it around. here's the AWS one for ref:

```
# maps release series of major.minor to cluster-api contract version
# the contract version may change between minor or major versions, but *not*
# between patch versions.
#
# update this file only when a new major or minor version is released
apiVersion: clusterctl.cluster.x-k8s.io/v1alpha3
releaseSeries:
  - major: 0
    minor: 4
    contract: v1alpha2
  - major: 0
    minor: 5
    contract: v1alpha3
  - major: 0
    minor: 6
    contract: v1alpha3
  - major: 0
    minor: 7
    contract: v1alpha4
```

Reply:
Actually amended and re-ordered these from lowest to higest version number to match what's in the AWS example.